src/configure/model/models.ts

```diff
@@ -52,6 +52,29 @@ export class Organization {
     isMSAOrg: boolean;
 }
 
+export class GitHubOrganization {
+    login: string;
```

Comment:
are all these properties mandatory for GitHub API and are we using all? If no, then just keep such fields.

Reply:
No, we are not using all the fields. I'll remove the unused fields.